tests/e2e/testcases/icm/sendMsg/suite.go

```diff
@@ -0,0 +1,522 @@
+			globalFlags := utils.GlobalFlags{
+				"local":             true,
+				"skip-update-check": true,
+			}
+
+			sendMsgFlags := utils.TestFlags{
+				"key": ewoqKeyName,
+			}
+
+			sendMessageArgs := []string{
+				subnetName,
+				cChain,
+				message,
+			}
+
+			output, err := utils.TestCommand(utils.ICMCmd, "sendMsg", sendMessageArgs, globalFlags, sendMsgFlags)
+			gomega.Expect(err).Should(gomega.BeNil())
+			gomega.Expect(output).Should(gomega.ContainSubstring(fmt.Sprintf("Delivering message \"%s\" from source blockchain \"%s\"", message, subnetName)))
+			gomega.Expect(output).Should(gomega.ContainSubstring(fmt.Sprintf("Waiting for message to be delivered to destination blockchain \"%s\"", cChain)))
+			gomega.Expect(output).Should(gomega.ContainSubstring("Message successfully Teleported!"))
+
+			commands.StopRelayer()
+		})
+
+		ginkgo.It("should send a message from subnet to subnet", func() {
+			commands.CreateSubnetEvmConfigNonSOV(subnetName, utils.SubnetEvmGenesisPath, false)
+			commands.DeploySubnetLocallyNonSOV(subnetName)
+
+			commands.CreateSubnetEvmConfigNonSOV(subnet2Name, utils.SubnetEvmGenesisPath, false)
+			commands.DeploySubnetLocallyNonSOV(subnet2Name)
+
+			// Deploy ICM to subnet1
+			_, err := commands.DeployICMContracts([]string{}, utils.TestFlags{
+				"key":        ewoqKeyName,
+				"blockchain": subnetName,
+			})
+			gomega.Expect(err).Should(gomega.BeNil())
+
+			// Deploy ICM to subnet2
+			_, err = commands.DeployICMContracts([]string{}, utils.TestFlags{
+				"key":        ewoqKeyName,
+				"blockchain": subnet2Name,
+			})
+			gomega.Expect(err).Should(gomega.BeNil())
+
+			// Deploy the relayer
+			_, err = commands.DeployRelayer(
+				[]string{
+					"deploy",
+					"--cchain",
+				},
+				utils.TestFlags{
+					"key":           ewoqKeyName,
+					"blockchains":   fmt.Sprintf("%s,%s", subnetName, subnet2Name),
+					"amount":        10000,
+					"cchain-amount": 10000,
+					"log-level":     "info",
+				})
+			gomega.Expect(err).Should(gomega.BeNil())
+
+			// Send a message
+			globalFlags := utils.GlobalFlags{
+				"local":             true,
+				"skip-update-check": true,
+			}
+
+			sendMsgFlags := utils.TestFlags{
+				"key": ewoqKeyName,
+			}
+
+			sendMessageArgs := []string{
+				subnet2Name,
+				subnetName,
+				message,
+			}
+
+			output, err := utils.TestCommand(utils.ICMCmd, "sendMsg", sendMessageArgs, globalFlags, sendMsgFlags)
+			gomega.Expect(err).Should(gomega.BeNil())
+			gomega.Expect(output).Should(gomega.ContainSubstring(fmt.Sprintf("Delivering message \"%s\" from source blockchain \"%s\"", message, subnet2Name)))
+			gomega.Expect(output).Should(gomega.ContainSubstring(fmt.Sprintf("Waiting for message to be delivered to destination blockchain \"%s\"", subnetName)))
+			gomega.Expect(output).Should(gomega.ContainSubstring("Message successfully Teleported!"))
+
+			commands.StopRelayer()
+		})
+
+		ginkgo.It("should send a message from subnet to subnet", func() {
```

Comment:
whats the diff between this and the one before

Reply:
Mistake. Will remove